posthog/templates/email/alert_anomaly.html

```diff
@@ -0,0 +1,10 @@
+{% extends "email/base.html" %} {% load posthog_assets %} {% block section %}
+<p>
+    Uh-oh, the <a href="{% absolute_uri alert_url %}">{{ alert_name }}</a> alert detected following anomalies for <a href="{% absolute_uri insight_url %}">{{ insight_name }}</a>:
```

Comment:
Not blocking but "uh-oh" feels unnecessarily negative. The alert could be a positive thing.

Reply:
Removed :+1: